### PR TITLE
Added One DS-Algo Question

### PR DESCRIPTION
**Pull request by kartikagarwal9**
I have added a question in q4.html with proper indentation and proper alignment. Consider it please.

### CODE REVIEW

File: ds-and-algo/answers/q4.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    </div>
+
+    <div>
+      <p>
+        <small>
+          <a href="../../index.html">Home</a>
+          &gt;
+          <a href="../index.html"> Data Structures and Algorithms Questions</a>
+          &gt; How to Reverse a Linked List ?
+          <!-- Your Question Title -->
```
Comment (dakshkhetan) [inline ds-and-algo/answers/q4.html:26]: Insert your question title here too.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

File: ds-and-algo/answers/q4.html (hunk 1 above)
Comment (kartikagarwal9) [reply ds-and-algo/answers/q4.html:26]: It's already their bro, I just forgot to remove the comment, the question is there in the 25th line of code. I hope I cleared your doubt

Comment (dakshkhetan) [reply ds-and-algo/answers/q4.html:26]: Oops, my bad.
Please remove the comment then.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
@kartikagarwal9 I just merged someone else's PR (Q4). So, please fetch the updated code (`git pull upstream master`) and resolve the conflicts (i.e. adding Q5 instead of Q4).

**Pull request closed without merging**